src/main/scala/com/typesafe/sbt/site/util/SiteHelpers.scala

```diff
@@ -44,7 +44,10 @@ object SiteHelpers {
       ))
 
   def watchSettings(config: Configuration): Seq[Setting[_]] =
-    Compat.watchSettings(config)
+    Compat.watchSettings(ThisScope.in(config))
```

Comment:
I hope `config: Configuration == ThisScope.in(config)`

Reply:
Yes, `key in config == key in ThisScope.in(config)`